### DISCUSSION TITLE
求激活码

### DISCUSSION THREAD

**Discussion by zhangbei59** [LFTEC/LearningFucker#41] [General]
------------------------------License------------------------------
WYR7qV+5s4i8LY/gXFKcrRP5wZQvuh/6SM3K219papoc12f/HJ0/NkllJsqpzcpRijo
eJpeWNyGOSySLrAf9JaGZ9fQmOdSeKfjkW69E9xGwdjKLGCa0oIlbwX8GpPxCeg4LIC
86HVbeBUEJUqA+XBYArVMGqtdnIqEEEgftrj5YsNKmEbQ8QAt7SAsNWDmr7HxNktGKg
ZvxnXZTfKykG+RqTIKAUPWL/Rs+X+mh3Zc1Zq6/rcsIdwN5LgnPliENEjRRxhtP9FhA
hpNr6Fhi+spwKOyOIxpdOjfaIYZDdhmbrRchq3v8lwjQxpqKbzAoqP0czzBRw17JyEu
xIE2zAiHJDnVbKwhmwSL0nP2itaRJJqUAUSP/lXBhb8KwI4iv+w4Cd1aOnYnhgE8/9D
/ioNE3l8QZOMBRSZ5fWp5CjGm6YFmExgpnbbNIL5RC1X8m+xZXcDx6F2juRICZX4h2C
s1VN4GfneM+GpBqExedbhYfdF1A1SKnZMoxFfgryiqdMqFbcdWTA8QwwVYNOVGQds+Y
xU3f2DyV2B0TFORzQHlPimZKP69a3IOD8b/1BeZUKUeHKNSmAy5dOKZC6h2uENxN/r1
z3YDN2L0IA0/obavvZvPuHCmoC5C6uoOVpMyfWZcpz9b10XFw3W3ftWYsr8ulYMnjRo
aDip851xE4e+g3VqWUIx+BKfVdgKlGQmkSM2KVB+Bw8iAr34EMUCV9twRCMDbwDVGMF
sFTOx7p+DQfnU1EOdzm9sQVm9V5MgRzTq8OgCHGCl+QzEaCj6vNbtWJn7rPbtUCl/HI
dViOV3mJj4Li5r49Uc3lzsPofABrV0yMymypegqymgCYGoybsmwwI/GVa3qUMkH0JfO
tomKxDyut1dvrEtD9qlzoGs9un4e1MfI+QBIm2MQGBwy+M1ohi40NPbqUaOQxnnzIrR
DfZNbUnH7/Xko3kQJH/P719vwnNlQv9VAjVztpEz+D/iD79Qiit+ylXHexUuweet1UU
JBhUOrGkeEIfBr4QBmcTgh4REg3GVZb1WC28zxAbKScEpl3r8E55AFiXTWr5rW/LmjU
lYqOX4XHIhjG4QCvTRwDtFserhF10/8NyoUYv8L73dTPaK9mEdvQq48y64G3SZM5Ues
PvEK7xbFLd76wvovTWWQb/DLtfJRxy3cVpx7JbESQK7rWzVMiB3WRUP9smyXDMM9PzG
dnwjwUc2V3IiEEqJSn0x4JuZEwItNtqLNyldgBN5rB7yVdwf6MNIq4FWaGp6L5VBXDm
31ZfoShTlJQsS0F5i6ofG6dIZdjvHwgaqwbvhTZ0uNpZCsuPyls5tqZIxHj0XwN6lz7
D07xuAu02d+3/fuNWV3+uHMncg==
------------------------------License------------------------------

**Comment by LFTEC** [comment]
jvfXHCRDhbGoOhx0o5L8bJ4rMYeGfp/2/VDleSBLo/neLLa0s/y27oZu0+y0C20t0nmRDLU4CKIwdQpGiGVA4Phrk8ZKV1GrqyFJhVuDhglO7w/kxTgJN75Lqnd+yePum81EJ1hFPBy0XKz4sAW0d2yDm9UYCGPbi0pRLizLJec9qozuOgi63QJdoaGL8W77Sm+6c0Ra8Ght8syprywvXE2GZPjSjSGsFkipDbfKh7TehUqucgIbODzQ6SG3Wnfzf6DafN8hk5g/tqaX8F9DbOiGB5k3NVNN0JezdxsIRSNUjdD0/Go+VIJVPXEJRaT40+ip6EFEd0XGw+c46HYqkwwEXnIBuTAlltxQmH12uY7m5e/XKLNp+3JIW807nT5RNzLhJPueivdm94QItsVnBk+IbP7F1rJ6SgTRtP9kYAbdsKVSfzqLt5uGxo3yys8NkmgnVHmgcseGrP8M76qYSADR5heUAdMenJakgyA9xcZnfEmUuFVZzIEyFUaMPbJMOUtX9SGDAQbY/66Lg4afDoh43QfwXfdTBzc74okLRL1GNlONyQcJUD9KvYiZjl/w1miMsoPouJv24F2k859N1OuoXCnM1kOXKz4sSOZ1MPSDGqvQVoCZ0BeeusMd1H/Ew2MzegYkNejW4qiFvVH78GlNRvzswoo1MyxwXpaGwvpqGO57C9nepmwKFFP35idv/Gwcuwd5Fql+gy7Ha+gXNpvKewMxTT5YyinEbLqHAyHAiC8tGWDoqXLcLOxRJDeqb20MSpczT1sqrQwRfsRmhWqEWOAGjSP5Y9dljdPAPdjP0XSd8TPps0zoevO5Q7wRZytoh8tcwlBaTXRgZCd/H1RRkmz01jWOeaq6j3Z1e0wz7jfk10plt5hwQGdWq+mVrQXmtVGPr6MgAzDZipmpA3qdvZMGblPJWxssx8kNaUK/MIz6uEwqQEIASO8IG/dCyZo2btS8Lq2ZDNNuENLPtbcbwAPtgoAvVshnLyoT2JhKqgLvmlVPxABpAd9WF1ykbDida0Yu0sC0lXy6GyOU22gTIiqKffm1ipzuUDMfsWB7AxzilwBJuj+tO07N8ORbMaozaWH2U40WeV3LI5aU5te06x02OWOnEQDvwyjCwP+9XuXm6F6cW8FMFijoWYGwb8n2AlyqoYMB05xlf7iXaTCBlTGqCTg29lphFp3eBZEwz5CJn7Qc54twzmlql3PPcvBBt1mfctpyvrRsEUQHJ4zZj27hNyvBO1V9UFNQyT7QXw/FxUuhn/rJ+gusBFTbHT92FBZu+S4awxZD1Sq4gYadhQGmO+kBSv+VgiefH8wIA9mjrxwZyptO+xd/WL+aaUAHNeAnHXcI/NQE6dGUmg==

**Reply by zhangbei59** [reply]
感谢大佬！
